extensions/amp-access/0.1/amp-access-client.js

```diff
@@ -126,8 +132,13 @@ export class AccessClientAdapter {
   }
 
   /** @override */
+  isPingbackEnabled() {
+    return this.isPingbackEnabled_;
+  }
+
+  /** @override */
   pingback() {
-    const promise = this.context_.buildUrl(this.pingbackUrl_,
+    const promise = this.context_.buildUrl(dev().assert(this.pingbackUrl_),
```

Comment:
Is the build process smart to strip this correctly? Also curious why is this a `dev.assert`?